File: API/Models/AddressModel.cs
namespace ModelService
{
    public class AddressModel
    {
        public int? AddressId { get; set; }
        public int CustomerId { get; set; }
        public string? Address { get; set; }
        public string? City { get; set; }
        public string? State { get; set; }
        public string? Country { get; set; }
        public string? PostalCode { get; set; }
        public string? Phone { get; set; }
        public DateTime? CreatedAt { get; set; }
        public bool? IsDeleted { get; set; }
        public bool IsPrimary { get; set; }
    }
}

File: UI/Angular UI/src/app/shared/models/address.model.ts
export interface AddressModel {
    addressId?: number;
    customerId?: number;
    address?: string;
    city?: string;
    state?: string;
    country?: string;
    postalCode?: string;
    phone?: number;
    createdAt?: Date;
    isDeleted?: boolean;
    isPrimary?: boolean;
}
